examples/question_answering.py

```diff
@@ -124,5 +124,10 @@ def question_answering():
         out_filename="predictions.json"
     )
 
+    # To get evaluate the model's performance on the SQuAD dev set, run the farm/squad_evaluation.py
```

Comment:
Can you make the value / use case of this clearer? Why should they consider doing this and not just take the FARM metrics produced during training? -> "To get metrics that are comparable to the SQuAD leaderboard and most other frameworks, you can now run the "official" squad eval script, which produces word level metrics and handles the no_answer option slightly different"

Reply:
Yup, this comment has been improved in the latest commit